composer/utils/__init__.py

```diff
@@ -23,6 +23,7 @@
     'ObjectStore',
     'ObjectStoreTransientError',
     'LibcloudObjectStore',
+    'S3ObjectStore'
     'MissingConditionalImportError',
```

Comment:
Could you add `SFTPObjectStore` as well? 🙏

Reply:
done